circe.el

```diff
@@ -1489,6 +1500,20 @@ PATTERNS should be the list of regular expressions."
           (throw 'return t)))
       nil)))
 
+(defun circe-tracking-get-face (buffer)
+  "Return face for a given buffer. If the buffer has already some face
+specified, use it. This covers channels where our name was mentioned. Otherwise
+decide whether the buffer is for a channel, query or else and use a face based
+on this."
+  (with-current-buffer buffer
+    (cond ((get-text-property 0 'face buffer)
+           (get-text-property 0 'face buffer))
```

Comment:
You can elide this line, if there's only a truthy condition, it evaluates to its result.

Reply:
Thank you, I wasn't aware that it is possible. Updated.